fsthttp/backend.go

```diff
@@ -311,6 +311,52 @@ func (b *BackendOptions) PoolConnections(poolingOn bool) *BackendOptions {
 	return b
 }
 
+// HttpKeepaliveTime configures how long to allow HTTP connections to remain
+// idle in a connection pool before it should be considered closed.
+func (b *BackendOptions) HttpKeepaliveTime(time time.Duration) *BackendOptions {
+	b.abiOpts.HttpKeepaliveTime(time)
+	return b
+}
+
+// TcpKeepaliveEnable sets whether or not to use TCP keepalives to try to
+// maintain the connetion to the backend.
+func (b *BackendOptions) TcpKeepaliveEnable(enable bool) *BackendOptions {
```

Comment:
Naming: `TCP` vs `Tcp`

Reply:
random hit says yes: https://pkg.go.dev/github.com/m-lab/tcp-info/tcp uses TCP

Reply:
Fixed in 107ec31!